src/main/java/com/homeaway/dropwizard/bundle/resilience4j/configuration/CircuitBreakerConfiguration.java

```diff
@@ -35,56 +38,67 @@
-     * then at least 100 calls must be evaluated, before the failure rate can be calculated. If only 99 calls have been evaluated the
-     * CircuitBreaker will not trip open even if all 99 calls have failed. The size must be greater than 0. Default size is 100.
-     * <br>
-     * See also {@link io.github.resilience4j.circuitbreaker.CircuitBreakerConfig.Builder#ringBufferSizeInClosedState}.
-     */
-    private int ringBufferSizeInClosedState = 100;
-
     /**
      * Configures the wait duration which specifies how long the CircuitBreaker should stay open, before it switches to half open.
      * Default value is 60000 milliseconds.
      * <br>
      * See also {@link io.github.resilience4j.circuitbreaker.CircuitBreakerConfig.Builder#waitDurationInOpenState}.
      */
+    @JsonProperty
     private Duration waitDurationInOpenState = Duration.seconds(60);
 
     /**
      * Enables automatic transition from OPEN to HALF_OPEN state once the waitDurationInOpenState has passed.
      * Default value is {c}true{/c}
      * See also {@link io.github.resilience4j.circuitbreaker.CircuitBreakerConfig.Builder#enableAutomaticTransitionFromOpenToHalfOpen}.
      */
+    @JsonProperty
     private Boolean enableAutomaticTransitionFromOpenToHalfOpen = true;
 
-    public CircuitBreakerConfig.Builder toResilience4jConfigBuilder() {
-        CircuitBreakerConfig.Builder builder = CircuitBreakerConfig.custom()
-                                          .waitDurationInOpenState(java.time.Duration.ofNanos(this.waitDurationInOpenState.toNanoseconds()))
-                                          .ringBufferSizeInClosedState(this.ringBufferSizeInClosedState)
-                                          .ringBufferSizeInHalfOpenState(this.ringBufferSizeInHalfOpenState)
-                                          .failureRateThreshold(this.failureRateThreshold);
+    @JsonProperty
+    private int minimumNumberOfCalls = CircuitBreakerConfig.DEFAULT_MINIMUM_NUMBER_OF_CALLS;
+
+    @JsonProperty
+    private int permittedNumberOfCallsInHalfOpenState = CircuitBreakerConfig.DEFAULT_PERMITTED_CALLS_IN_HALF_OPEN_STATE;
+
+    @JsonProperty
+    private int slidingWindowSize = CircuitBreakerConfig.DEFAULT_SLIDING_WINDOW_SIZE;
+
+    @JsonProperty
+    private CircuitBreakerConfig.SlidingWindowType slidingWindowType = CircuitBreakerConfig.DEFAULT_SLIDING_WINDOW_TYPE;
+
+    @JsonProperty
+    private Duration slowCallDurationThreshold = Duration.seconds(CircuitBreakerConfig.DEFAULT_SLOW_CALL_DURATION_THRESHOLD);
+
+    @JsonProperty
+    private float slowCallRateThreshold = CircuitBreakerConfig.DEFAULT_SLOW_CALL_RATE_THRESHOLD;
+
+    @JsonProperty
+    private boolean writableStackTraceEnabled = CircuitBreakerConfig.DEFAULT_WRITABLE_STACK_TRACE_ENABLED;
 
-        if (enableAutomaticTransitionFromOpenToHalfOpen) {
-            builder.enableAutomaticTransitionFromOpenToHalfOpen();
-        }
+    @JsonProperty
+    private Class[] ignoreExceptions = new Class[0];
```

Comment:
Can we match the signature from resilience4j? (i.e. `Class<? extends Throwable>[]`)

The same for the property below.